src/components/icons/components/action/index.ts

```diff
@@ -7,3 +7,4 @@ export { default as CollapseIcon } from './CollapseIcon';
 export { default as CalendarIcon } from './CalendarIcon';
 export { default as InfoIcon } from './InfoIcon';
 export { default as HistoryIcon } from './HistoryIcon';
```

Comment:
Should this also be exported from src/index.ts?

Reply:
It is ;)